packages/account/src/Constants/api-token-card-details.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+
+export const getApiTokenCardDetails = () => [
```

Comment:
**NIT**: Now that we are using `<Localize/>` can this be a const variable rather than a function?